src/ape/contracts/base.py

```diff
@@ -208,12 +208,14 @@ def serialize_transaction(self, *args, **kwargs) -> TransactionAPI:
         )
 
     def __call__(self, *args, **kwargs) -> ReceiptAPI:
+        raise_on_fail = kwargs.pop("raise_on_fail", True)
```

Comment:
Just fixed this in 12d9cd20 because only `get_receipt()` and `await_confirmations()` default to `False`, everything else defaults to `True` so that should include this as well